virtualcontext.js

```diff
@@ -8,14 +8,13 @@
 const Types = require("./types");
 const Context = require("./context");
 
-function VirtualContext(id, endpoint=null, parent=null)
+function VirtualContext(id, src=null, parent=null)
 {
 	if (arguments[0] && typeof arguments[0] === "object")
```

Comment:
Is `virtualSrc` a required property? If yes, should it not be its own property instead of being within the `properties` property?

Reply:
In fact this was the first implementation. But since our intention is not to expand the conceptual model and, therefore, treat this entity as just any context with auxiliary characteristics, we insert it as a property. Furthermore, it avoids the creation of a new url to be handled by rdf2hk.

Reply:
In the current implementation `new VirtualContext({properties: {}})` is a valid statement. The new instance would not pass the `isValid()` test...

Reply:
Ok, I think I got the pattern... It allows clients to instantiate invalid objects; if another piece of code requires valid instances it should check calling `.isValid()`.